fh-messaging/lib/fh_filter_man.js

```diff
@@ -176,7 +176,6 @@ var filterOneAsync = function(message, cb) {
 function FilterManager(config, logger) {
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Rename this 'FilterManager' function to match the regular expression ^[a-z][a-zA-Z0-9]*$ [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3AS100)